pubsub/jetstream/jetstream.go

```diff
@@ -88,8 +96,26 @@ func (js *jetstreamPubSub) Features() []pubsub.Feature {
 }
 
 func (js *jetstreamPubSub) Publish(req *pubsub.PublishRequest) error {
-	js.l.Debugf("Publishing topic %v with data: %v", req.Topic, req.Data)
-	_, err := js.jsc.Publish(req.Topic, req.Data)
+	var opts []nats.PubOpt
+	var msgId string
```

Comment:
I couldn't find a way to output the PubOpt's in the debug logs, so I've opted to store anything printed out in a separate variable.